tracer/build/_build/Build.Steps.cs

```diff
@@ -194,6 +195,7 @@ partial class Build
     Target CompileNativeSrcLinux => _ => _
         .Unlisted()
         .After(CompileManagedSrc)
+        .DependsOn(CppCheckNativeSrcLinux)
```

Comment:
I don't think we should add this CppCheck here. We don't want to run this _every_ time we build, as it will increase cycle time. Instead, add `CppCheckNativeSrcLinux` to the ci build?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3483/commits/87ff896d3f3fafb48844ebbbf0407d22bac7452a